### DISCUSSION TITLE
通过helm 部署，数据库初始化老是失败，图形界面的dashboard，500

### DISCUSSION THREAD

**Discussion by sscash** [hhyo/Archery#2737] [General]
mysql 版本8.0 初始化的时候提示。
![图片](https://github.com/user-attachments/assets/93995a08-0e3d-44a1-b8a2-8e588b11ebb4)
![图片](https://github.com/user-attachments/assets/9de94264-024a-4e94-bee2-99aa09b15c31)
![图片](https://github.com/user-attachments/assets/62c2fbe2-0f71-4e9a-bd53-b20f06b0a713)
日志：
![图片](https://github.com/user-attachments/assets/00d14622-e808-4b13-84c9-c8f066f43a8e)

**Comment by visla-xugeng** [comment]
Error code: mysql_slow_query_review_history. You need to manually create this table in Archery DB. https://github.com/hhyo/archery/wiki/docker#%E5%90%AF%E5%8A%A8

```
python3 manage.py dbshell<src/init_sql/mysql_slow_query_review.sql
```
